dist/linux/app.desktop

```diff
@@ -16,7 +18,8 @@ X-TerminalArgTitle=--title=
 X-TerminalArgAppId=--class=
 X-TerminalArgDir=--working-directory=
 X-TerminalArgHold=--wait-after-command
+DBusActivatable=true
 
 [Desktop Action new-window]
 Name=New Window
-Exec=ghostty
+Exec=ghostty --launched-from=desktop
```

Comment:
This one should use absolute path as well.

Reply:
Ah, good catch! Fixed.